src/Core/Models/Response/OrganizationDomainSsoDetailsResponse.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+namespace Bit.Core.Models.Response
+{
+    public class OrganizationDomainSsoDetailsResponse
+    {
+        public bool SsoAvailable { get; private set; }
+        public string DomainName { get; private set; }
+        public string OrganizationIdentifier { get; private set; }
+        public bool SsoRequired { get; private set; }
+        public DateTime? VerifiedDate { get; private set; }
+    }
```

Comment:
Mmm I don't think this will work with `private set` cause the json deserializer won't be able to set the values (unless private setting is activated). Why do you need the properties to be `private`?

Reply:
It was copy paste from the server i didn't even realised, thanks. Will change to public.